keyboards/mechlovin/zed65/rgb/info.json

```diff
@@ -0,0 +1,173 @@
+    "url": "https://mechlovin.studio/", 
+    "maintainer": "mechlovin",
+    "keyboard_name": "Zed65 RGB",
+    "usb": {
+        "vid": "0x4D4C",
+        "pid": "0x6510",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "nkro": true,
+        "rgblight": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "rows": ["B11", "B10", "B2", "B1", "B0"],
+        "cols": ["A5", "A4", "A3", "A7", "B12", "A15", "B3", "B4", "B5", "B6", "B7", "A8", "B15", "B14", "B13"]
+    },
+    "build": {
+       "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "rgb_matrix": {
+        "animations": {
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "rainbow_moving_chevron": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_rain": true,
+            "pixel_flow": true,
+            "pixel_fractal": true,
+            "typing_heatmap": true,
+            "digital_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "default": {
+            "animation": "cycle_all"
+        },
+        "driver": "is31fl3741",
+        "sleep": true
+    },
+    "rgblight": {
+        "led_count": 9,
+        "sleep": true,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "ws2812": {
+        "pin": "A6",
+        "driver": "pwm"
+    },
+    "community_layouts": ["65_ansi_blocker_tsangan"],
```

Comment:
Looking at image provided in `readme.md`, could the following Community Layouts be added to `community_layouts` and `layouts`?
- 65_ansi
- 65_ansi_split_bs
- 65_ansi_blocker
- 65_ansi_blocker_split_bs
- 65_ansi_blocker_tsangan_split_bs
- 65_iso
- 65_iso_split_bs
- 65_iso_blocker
- 65_iso_blocker_split_bs
- 65_iso_blocker_tsangan
- 65_iso_blocker_tsangan_split_bs

Alternatively, If you could identify matrix co-ordinates (e.g. `1, 13`) for the following key positions, I can submit code suggestions for named Community Layouts:
- Split 1u backspace marked `DEL` on PCB
- ISO Enter
- ISO Hash (3rd row, right of `'`)
- ISO Backslash (4th row, right of 1.25u Left Shift)